maintainers/team-list.nix

```diff
@@ -501,6 +501,7 @@ with lib.maintainers;
     members = [
       euank
       marcusramberg
+      rorosen
```

Comment:
Usually we keep alphabetical order, just in case it grows large.

Reply:
I see, it's alphabetical now.